cavern/src/main/java/org/opencadc/cavern/nodes/FileSystemNodePersistence.java

```diff
@@ -144,16 +149,27 @@ public class FileSystemNodePersistence implements NodePersistence {
     private final CavernConfig config;
     private final boolean localGroupsOnly;
 
+    // Set of default properties for nodes.  This will be built from configuration.
+    private final List<NodeProperty> defaultProperties = new ArrayList<>();
```

Comment:
do we anticipate other default properties? I think it is very unlikely...

I would have a single explicit `NodeProperty DEFAULT_QUOTA` to start and make it a list later.